### PR TITLE
Show Hovering Translations

### PR DESCRIPTION
**Pull request by Brian-Bartels**
I grabbed most of the code from https://github.com/intellism/vscode-comment-translate. However, I did a LOT of stripping down of that codebase. There is a lot of stuff here so please reach out with any questions.

BIG CHANGES:
Now we essentially have 2 projects, the main one which lives mostly in extension.js, and the server which lives in server. NPM install will still work because I added a post install step

Since the server is written in Typescript, you have to compile after every change. This could be done in a .vscode run file. I just ran "npm run compile" before making changes.

The above change COULD cause issues with your deployment process. I'm happy to work on that with you. Since this is my last planned PR into this repo, I don't want to invest in CI pipeline.

KNOWN ISSUES:
If the user doesn't have a preferred language set, the default is English. This is somewhat silly but as soon as the user sets the preferred language the hover works great.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,11 +17,7 @@
         "Formatters"
     ],
     "activationEvents": [
-        "onCommand:extension.translateText",
-        "onCommand:extension.setPreferredLanguage",
-        "onCommand:extension.translateTextPreferred",
-        "onCommand:extension.translateLinesUnderCursor",
-        "onCommand:extension.translateLinesUnderCursorPreferred"
+        "*"
```
Comment (Brian-Bartels) [inline package.json:20]: This will activate extension on startup of VSCode

**Comment by funkyremi** [comment]
Hi :-) Thanks for this bug PR.
I tried this new feature but I cannot make it work. Could you please provide examples/gif?

**Comment by Brian-Bartels** [comment]
> Hi :-) Thanks for this bug PR.
> I tried this new feature but I cannot make it work. Could you please provide examples/gif?

Hm, did you run "npm run compile" before starting the extension from the debugger? The typescript server needs to be translated to JS in an out folder. Otherwise extension.js is referencing a non-existent file. I'll get some gifs too

**Comment by funkyremi** [comment]
Thank you :D I made small changes. The new version is released! 1.4.8

**Pull request merged**